src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Folding/FoldingRangeEndpoint.cs

```diff
@@ -55,7 +55,7 @@ public TextDocumentIdentifier GetTextDocumentIdentifier(FoldingRangeParams reque
 
         var requestParams = new RazorFoldingRangeRequestParam
         {
-            HostDocumentVersion = documentContext.Version,
+            HostDocumentVersion = documentContext.Snapshot.Version,
```

Comment:
I can't tell from diff why we need to get version from Snapshot

Reply:
Simply because that's where it is now. It used to be on `DocumentContext` by way of `DocumentVersionCache`, but the version cache has been deleted and each document snapshot is responsible for its own version number now.